pkg/webui/components/status/story.js

```diff
@@ -15,16 +15,22 @@
 import React from 'react'
 import bind from 'autobind-decorator'
 import { storiesOf } from '@storybook/react'
+import { defineMessages } from 'react-intl'
 
 import Status from '.'
 
 const containerStyle = {
-  width: '100px',
+  width: '120px',
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'space-between',
 }
 
+const m = defineMessages({
```

Comment:
There is no reason to define custom translated messages just for storybook. If the `<Status />` component does not accept raw strings, it is a bug and should fixed.

Reply:
yep there was an issue with calling `intl.formatMessage` with a string as a parameter instead of a message, now this case is handled correctly.

Reply:
👌